scheduler/scheduling/evaluator/evaluator_network_topology.go

```diff
@@ -0,0 +1,282 @@
+/*
+ *     Copyright 2020 The Dragonfly Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package evaluator
+
+import (
+	"math/big"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/montanaflynn/stats"
+
+	logger "d7y.io/dragonfly/v2/internal/dflog"
+	"d7y.io/dragonfly/v2/pkg/math"
+	"d7y.io/dragonfly/v2/pkg/types"
+	"d7y.io/dragonfly/v2/scheduler/networktopology"
+	"d7y.io/dragonfly/v2/scheduler/resource"
+)
+
+const (
+	// Finished piece weight.
+	networkTopologyPartFinishedPieceWeight float64 = 0.2
```

Comment:
networkTopologyPart naming is wrong

Reply:
Now I change it to networkTopologyBasedAlgorithmFinishedPieceWeight. How about it?